iodata/formats/gamess.py

```diff
@@ -0,0 +1,145 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+"""GAMESS punch file format."""
+
+
+import numpy as np
+
+from ..docstrings import document_load_one
+from ..utils import angstrom, LineIterator
+
+
+__all__ = []
+
+
+PATTERNS = ['*.dat']
+
+
+def _read_data(lit: LineIterator) -> tuple:
+    """Extract ``title``, ``symmetry`` and ``symbols`` from the punch file."""
+    title = next(lit).strip()
+    symmetry = next(lit).split()[0]
+    # The dat file only contains symmetry-unique atoms, so we would be incapable of
+    # supporting non-C1 symmetry without significant additional coding.
+    if symmetry != "C1":
+        raise NotImplementedError(f"Only C1 symmetry is supported. Got {symmetry}")
```

Comment:
I am not sure how to test this without adding a new test file (which is not desirable). Any suggestions @tovrstra?

Reply:
It is also impractical to add files at this stage. We would need to get in touch with someone who is at least a casual GAMESS user to generate good files. That would allow us to further elaborate on this format.